### ISSUE TITLE
Linux mono version crashes on Settings

### ISSUE THREAD

**Issue by youpet2000** [gitextensions/gitextensions#2845]
This is similar to a previously closed issue.  I am running the mono version on Ubuntu 10LTS.  Whenever I select Settings via any method, the app crashes, shuts down without error message.  I tried older versions that I know previously worked and they now do the same thing.  Seems like maybe I lost a config file when I replaced app folder with new version un-zipped folder.

**Comment by JerryLumpkins** [comment]
I use git-extensions 2.48.05 daily on Ubuntu 15.04, and used to use it on 14.04, and 14.10. I've never had the setting window crash. I was curious about the state of Ubuntu 10LTS, and just checked. It was has reached its end of life as of April 30th 2015: http://fridge.ubuntu.com/2015/03/18/ubuntu-10-04-lucid-lynx-reaches-end-of-life-on-april-30-2015/

**Comment by Carbenium** [comment]
Can't reproduce on Mint 17.3. I guess this can be closed.

**Comment by bking** [comment]
I know this was closed, but I just had the same issue today on Debian Stretch. I'll paste the stacktrace below where settings crashed. I installed 'mono-complete' (128 packages- overkill maybe? I'm new to mono...) and then ran again, and the problem went away.

`libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: cHRM chunk does not match sRGB
System.BadImageFormatException: Could not resolve field token 0x04001303
File name: 'GitUI'
  at GitUI.CommandsDialogs.SettingsDialog.SettingsPageBase.Create[T] (GitUI.CommandsDialogs.SettingsDialog.ISettingsPageHost aPageHost) [0x00005] in <b4af8474658e460f94a15619a11716c1>:0 
  at GitUI.CommandsDialogs.FormSettings..ctor (GitUI.GitUICommands aCommands, GitUI.CommandsDialogs.SettingsDialog.SettingsPageReference initalPage) [0x0016d] in <b4af8474658e460f94a15619a11716c1>:0 
  at (wrapper remoting-invoke-with-check) GitUI.CommandsDialogs.FormSettings:.ctor (GitUI.GitUICommands,GitUI.CommandsDialogs.SettingsDialog.SettingsPageReference)
  at GitUI.CommandsDialogs.FormSettings.ShowSettingsDialog (GitUI.GitUICommands uiCommands, System.Windows.Forms.IWin32Window owner, GitUI.CommandsDialogs.SettingsDialog.SettingsPageReference initalPage) [0x00021] in <b4af8474658e460f94a15619a11716c1>:0 
  at GitUI.GitUICommands+<>c__DisplayClass381_0.<StartSettingsDialog>b__0 () [0x00000] in <b4af8474658e460f94a15619a11716c1>:0 
  at GitUI.GitUICommands.DoActionOnRepo (System.Windows.Forms.IWin32Window owner, System.Boolean requiresValidWorkingDir, System.Boolean changesRepo, GitUIPluginInterfaces.GitUIEventHandler preEvent, GitUIPluginInterfaces.GitUIPostActionEventHandler postEvent, System.Func`1[TResult] action) [0x0002d] in <b4af8474658e460f94a15619a11716c1>:0 
  at GitUI.GitUICommands.StartSettingsDialog (System.Windows.Forms.IWin32Window owner, GitUI.CommandsDialogs.SettingsDialog.SettingsPageReference initialPage) [0x00028] in <b4af8474658e460f94a15619a11716c1>:0 
  at GitUI.CommandsDialogs.FormBrowse.SettingsClick (System.Object sender, System.EventArgs e) [0x0000b] in <b4af8474658e460f94a15619a11716c1>:0 
  at System.Windows.Forms.ToolStripItem.OnClick (System.EventArgs e) [0x0001c] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripMenuItem.OnClick (System.EventArgs e) [0x000a5] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripMenuItem.HandleClick (System.Int32 mouse_clicks, System.EventArgs e) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripItem.FireEvent (System.EventArgs e, System.Windows.Forms.ToolStripItemEventType met) [0x00061] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at (wrapper remoting-invoke-with-check) System.Windows.Forms.ToolStripItem:FireEvent (System.EventArgs,System.Windows.Forms.ToolStripItemEventType)
  at System.Windows.Forms.ToolStrip.OnMouseUp (System.Windows.Forms.MouseEventArgs mea) [0x00057] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripDropDown.OnMouseUp (System.Windows.Forms.MouseEventArgs mea) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control.WmLButtonUp (System.Windows.Forms.Message& m) [0x0007e] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control.WndProc (System.Windows.Forms.Message& m) [0x0016f] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ScrollableControl.WndProc (System.Windows.Forms.Message& m) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStrip.WndProc (System.Windows.Forms.Message& m) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripDropDown.WndProc (System.Windows.Forms.Message& m) [0x0001a] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control+ControlWindowTarget.OnMessage (System.Windows.Forms.Message& m) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.WndProc (System.Windows.Forms.Message& m) [0x0000b] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.NativeWindow.WndProc (System.IntPtr hWnd, System.Windows.Forms.Msg msg, System.IntPtr wParam, System.IntPtr lParam) [0x0008e] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
`
